cli/cmd/cloud_account.go

```diff
@@ -12,9 +14,75 @@ var (
 		Short:   "Manage cloud accounts",
 		Long:    "Manage cloud account integrations with Lacework",
 	}
+
+	// used by cloud account list to list only a single type of cloud account
+	cloudAccountType string
+
+	// cloudAccountsListCmd represents the list sub-command inside the cloud accounts command
+	cloudAccountListCmd = &cobra.Command{
+		Use:   "list",
+		Short: "List all available cloud account integrations",
+		Args:  cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			var (
+				cloudAccounts api.CloudAccountsResponse
+				err           error
+			)
+
+			if cloudAccountType != "" {
+				caType, found := api.FindCloudAccountType(cloudAccountType)
+				if !found {
+					return errors.Errorf("unknown cloud account type '%s'", cloudAccountType)
+				}
+				cloudAccounts, err = cli.LwApi.V2.CloudAccounts.ListByType(caType)
+			} else {
+				cloudAccounts, err = cli.LwApi.V2.CloudAccounts.List()
+			}
+			if err != nil {
+				return errors.Wrap(err, "unable to get cloud accounts")
+			}
+
+			if cli.JSONOutput() {
+				return cli.OutputJSON(cloudAccounts.Data)
+			}
+
+			if len(cloudAccounts.Data) == 0 {
```

Comment:
You need to move this length check above the json output. Otherwise when there are no cloud accounts and the --json flag is set the output will be `null`